app/models/event.rb

```diff
@@ -0,0 +1,9 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User', foreign_key: 'creator_id'
+  has_many :attendances
+  validates :event_date, presence: true
+  default_scope -> { order(event_date: :asc) }
+
+  scope :upcoming, -> { where('event_date >= ?', Time.now.strftime('%Y-%d-%m')) }
+  scope :past, -> { where('event_date < ?', Time.now.strftime('%Y-%d-%m')) }
```

Comment:
There is a bug here because future events are being registered as past events, take a look:

```
irb(main):008:0> Event.past.last.event_date
  Event Load (0.5ms)  SELECT  "events".* FROM "events" WHERE (event_date < '2020-23-03') ORDER BY "events"."event_date" DESC LIMIT ?  [["LIMIT", 1]]
=> Tue, 21 Jul 2020
```

[`REQUIRED`]

Reply:
we fix that bug related to Time.now.strftime('%Y-%d-%m')